introduction.md

```diff
@@ -0,0 +1 @@
+Hello, where in teh world should my face be that would not give thios silly thing a thingy in he rear.4
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.